### ISSUE TITLE
Bug auto placed items not displayed

### ISSUE THREAD

**Issue by klemenoslaj** [tiberiuzuld/angular-gridster2#310]
Currently I am blocked from updating to `5.3.0` because of the single most important behavior of gridster(for me), which now behaves completely different.

Before to my understanding, on a fixed layout gridster tried to place all the gridster items to available space at the bottom, if `maxCols` was defined. Now items mysteriously do not appear(because of initial `display: none`).

### Reproducible setup
```
this.options = {
  gridType: GridType.Fixed,
  maxCols: 4,
};

this.dashboard = [
  {cols: 2, rows: 1, y: 0, x: 0},
  {cols: 2, rows: 2, y: 0, x: 2},
  {cols: 1, rows: 1, y: 0, x: 4},
  {cols: 1, rows: 1, y: 2, x: 5},
];
```

**Comment by tiberiuzuld** [comment]
Fixed in v5.3.1

Thanks

**Comment by klemenoslaj** [comment]
That was fast! Thanks 👍
